plugins/GSdx/GSSettingsDlg.h

```diff
@@ -64,6 +64,20 @@ class GSHacksDlg : public GSDialog
 	}
 };
 
+class GSOSDDlg : public GSDialog
+{
+	int transparency;
```

Comment:
Well, old code isn't necessarily good, and I say that as the person who didn't use the unique_ptr above. ;)

Personally I'd prefer just using a local variable when necessary (same applies to the other dialog sliders), but you can leave it as it is (well, you should at least prefix it with a `m_` so I don't search the code looking for where the variable is declared).